lib/ansible.js

```diff
@@ -143,6 +148,7 @@ AbstractAnsibleCommand.prototype.commonCompileParams = function(commandParams) {
   commandParams = this.addParam(commandParams, "limit", "l");
   commandParams = this.addParam(commandParams, "su", "U");
   commandParams = this.addPathParam(commandParams, "privateKey", "-private-key");
+  commandParams = this.addParam(commandParams, "vaultPasswordFile", "-vault-password-file");
```

Comment:
This should probably use `addPathParam()` instead of `addParam()`. However, when I tried using `addPathParam()`, I got this error

    ERROR! The vault password file /srv/ansible_api/"/srv/ansible/test.txt" was not found

`/srv/ansible_api/` is the working directory, and `/srv/ansible/test.txt` is the value I passed into the `vaultPasswordFile()` function. 

`addPathParam()` works when passing a filename into `privateKey()`, so I'm not sure why it doesn't work with `vaultPasswordFile()`.

Reply:
I think that ```addPathParam()``` is not working for the same reason of #28